### PR TITLE
Prepare 0.5 release

### PR DESCRIPTION
**Pull request by jl-wynen**

### CODE REVIEW

File: docs/about/release-notes.rst [hunk 1]
```diff
@@ -3,25 +3,20 @@
...
+------
 
 Features
 ~~~~~~~~
 
-Breaking changes
-~~~~~~~~~~~~~~~~
+* Added Utilities for unwrapping frames `#242 <https://github.com/scipp/scippneutron/pull/242>`_.
+* Added First draft of low-level utilities for loading NeXus files `#249 <https://github.com/scipp/scippneutron/pull/249>`_.
+* Multi-valued transforms are now loaded by :func:`scippneutron.load_nexus` `#267 <https://github.com/scipp/scippneutron/pull/267>`_.
```
Comment (Tom-Willemsen) [inline docs/about/release-notes.rst:14]: Might be worth clarifying where/how multi-valued transforms get loaded from (which is `NXlog` groups in the nexus file)?

Comment (jl-wynen) [reply docs/about/release-notes.rst:14]: Like this?
```.rst
* Multi-valued transforms are now loaded from ``NXlog``s in NeXus files by :func:`scippneutron.load_nexus` `#267 <https://github.com/scipp/scippneutron/pull/267>`_.
```

We need proper documentation of how we load files, what structure we assume, and how it is encoded in the output. But that is a separate issue for when we converge on a solution...

File: docs/about/release-notes.rst [hunk 2]
```diff
@@ -3,25 +3,20 @@
 Release Notes
 =============
 
-v0.5.0 (unreleased)
---------------------
+v0.5.0
+------
```
Comment (nvaytet) [inline docs/about/release-notes.rst:7]:
```suggestion
v0.5.0 (February 2022)
----------------------
```

File: docs/about/release-notes.rst [hunk 3]
```diff
@@ -3,25 +3,20 @@
 Release Notes
```
Comment (nvaytet) [inline docs/about/release-notes.rst:3]: Simon usually adds the `v0.6.0 (unreleased)` section before the release is made, but I think it's better to leave it out until after the release, as it looks cleaner in the docs pages for a specific release.

Comment (jl-wynen) [reply docs/about/release-notes.rst:3]: My thoughts exactly

**Review by Tom-Willemsen — APPROVED** [review]

**Review by nvaytet — APPROVED** [review]

**Pull request merged**